### PR TITLE
383 logo on readme is not displayed

### LINKED ISSUE #383: Logo on README is not displayed
## Subject of the issue

If the documentation request is approved, would you be willing to submit a PR? _(Help
can be provided if you need assistance submitting a PR)_

Yes

## Steps to reproduce

https://github.com/DeepRegNet/DeepReg

## Expected behaviour

It should be displayed.

## Actual behaviour

It is not displayed.

### PR DESCRIPTION
**Pull request by mathpluscode**
# Description

Fix master -> main related errors

This is a hotfix, so merge to master directly

Fixes #383 #371

## Type of change

What types of changes does your code introduce to DeepReg? _Put an `x` in the boxes that
apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Documentation Update (fix or improvement on the documentation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR.
If you're unsure about any of them, don't hesitate to ask. We're here to help! This is
simply a reminder of what we are going to look for before merging your code._

- [x] I have
      [installed pre-commit](https://deepreg.readthedocs.io/en/latest/contributing/setup.html)
      and formatted all changed files.
- [x] My commits' message styles matches
      [our requested structure](https://deepreg.readthedocs.io/en/latest/contributing/commit.html),
      e.g. `Issue #<issue number>: detailed message`.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by zacbaum** [comment]
@mathpluscode @YipengHu @NMontanaBrown - Since this "broke" the link to the educational challenge in the readme, will that also affect what was sent to MICCAI? Just a thought.

**Comment by mathpluscode** [comment]
> @mathpluscode @YipengHu @NMontanaBrown - Since this "broke" the link to the educational challenge in the readme, will that also affect what was sent to MICCAI? Just a thought.

good point! let me run the notebook to have a look, in the worst case we send a new email to explain.
(as using main instead of master is understandable change)

**Comment by mathpluscode** [comment]
> @mathpluscode @YipengHu @NMontanaBrown - Since this "broke" the link to the educational challenge in the readme, will that also affect what was sent to MICCAI? Just a thought.

The notebook should work as no `master` is detected in the notebook. Maybe we can still send an email.

**Comment by zacbaum** [comment]
> > @mathpluscode @YipengHu @NMontanaBrown - Since this "broke" the link to the educational challenge in the readme, will that also affect what was sent to MICCAI? Just a thought.
> 
> The notebook should work as no `master` is detected in the notebook. Maybe we can still send an email.

I meant link *to* the notebook - sorry that wasn't clear! Before it would've been .../master/... now it would be .../main/...

**Comment by mathpluscode** [comment]
> > > @mathpluscode @YipengHu @NMontanaBrown - Since this "broke" the link to the educational challenge in the readme, will that also affect what was sent to MICCAI? Just a thought.
> > 
> > 
> > The notebook should work as no `master` is detected in the notebook. Maybe we can still send an email.
> 
> I meant link _to_ the notebook - sorry that wasn't clear! Before it would've been .../master/... now it would be .../main/...

I corrected in README ;)

File: README.md [hunk 1]
```diff
@@ -48,6 +48,6 @@ consider
 ## MICCAI 2020 Educational Challenge
 
 The MICCAI Educational Challenge notebook can be accessed
-[here](https://github.com/DeepRegNet/DeepReg/blob/master/docs/Intro_to_Medical_Image_Registration.ipynb)
+[here](https://github.com/DeepRegNet/DeepReg/blob/main/docs/Intro_to_Medical_Image_Registration.ipynb)
 or run it on
-[Colab](https://colab.research.google.com/github/DeepRegNet/DeepReg/blob/master/docs/Intro_to_Medical_Image_Registration.ipynb).
+[Colab](https://colab.research.google.com/github/DeepRegNet/DeepReg/blob/main/docs/Intro_to_Medical_Image_Registration.ipynb).
```
Comment (mathpluscode) [inline README.md:53]: @zacbaum here ;)

**Comment by zacbaum** [comment]
> > > > @mathpluscode @YipengHu @NMontanaBrown - Since this "broke" the link to the educational challenge in the readme, will that also affect what was sent to MICCAI? Just a thought.
> > > 
> > > 
> > > The notebook should work as no `master` is detected in the notebook. Maybe we can still send an email.
> > 
> > 
> > I meant link _to_ the notebook - sorry that wasn't clear! Before it would've been .../master/... now it would be .../main/...
> 
> I corrected in README ;)

Yes I saw - but I meant the link we sent to MICCAI as part of the submission likely is now not working.

**Comment by NMontanaBrown** [comment]
I have emailed the student board about this change, giving them a new link replacing master with main, and hopefully we will hear back about it soon. @YipengHu @mathpluscode @zacbaum

**Review by YipengHu — APPROVED** [review]

**Pull request merged**